src/Discovery/reduxer.js

```diff
@@ -2,8 +2,10 @@ import { connect } from 'react-redux';
 import DiscoveryActionBar from './DiscoveryActionBar';
 
 const ReduxDiscoveryActionBar = (() => {
-  const mapStateToProps = (state) => ({
+  const mapStateToProps = (state, dispatch) => ({
     user: state.user,
+    discovery: state.discovery,
+    dispatch,
```

Comment:
this could work but a more standard way is to use the `mapDispatchToProps` for `connect` and pass in those dispatchable actions as props, see https://react-redux.js.org/using-react-redux/connect-mapdispatch